package.json

```diff
@@ -47,39 +47,43 @@
   },
   "homepage": "https://github.com/libp2p/js-interfaces#readme",
   "dependencies": {
-    "@types/bl": "^2.1.0",
+    "@types/bl": "4.1.0",
     "abort-controller": "^3.0.0",
     "abortable-iterator": "^3.0.0",
-    "chai": "^4.2.0",
+    "chai": "^4.3.4",
     "chai-checkmark": "^1.0.1",
     "debug": "^4.3.1",
-    "delay": "^4.4.0",
+    "delay": "^5.0.0",
     "detect-node": "^2.0.4",
     "dirty-chai": "^2.0.1",
-    "err-code": "^2.0.3",
+    "err-code": "^3.0.1",
     "it-goodbye": "^2.0.2",
     "it-length-prefixed": "^3.1.0",
     "it-pair": "^1.0.0",
     "it-pipe": "^1.1.0",
-    "it-pushable": "^1.4.0",
+    "it-pushable": "^1.4.2",
     "libp2p-crypto": "^0.19.0",
     "libp2p-tcp": "^0.15.0",
     "multiaddr": "^8.1.2",
-    "multibase": "^3.1.1",
-    "multihashes": "^3.1.1",
+    "multibase": "^4.0.2",
+    "multihashes": "^4.0.2",
     "p-defer": "^3.0.0",
     "p-limit": "^3.1.0",
     "p-wait-for": "^3.2.0",
     "peer-id": "^0.14.2",
     "protons": "^2.0.0",
-    "sinon": "^9.2.4",
+    "sinon": "^10.0.0",
     "streaming-iterables": "^5.0.4",
-    "uint8arrays": "^2.0.5"
+    "uint8arrays": "^2.1.3"
   },
   "devDependencies": {
-    "aegir": "^29.2.0",
+    "@types/debug": "^4.1.5",
```

Comment:
I'm assuming this is a devdep because it's an internal concern and we only want it to typecheck internal consistency and don't expose `debug` to users in any way form here. Is that correct?

Reply:
yes, that is correct!